aspnet-core/Menu/Context-Menu.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Context Menu | Menu | ASP.NET Core | Syncfusion
+title: Context Menu in ASP.NET Core Menu | Syncfusion
```

Comment:
Context Menu in ASP.NET Core | Syncfusion

Change the title as per above.

Reply:
changed